src/arrayMethodSort.js

```diff
@@ -3,9 +3,28 @@
 /**
  * Implement method Sort
  */
+const defaultCompare = (a, b) => {
+  return `${a}` < `${b}`
+    ? -1
+    : `${a}` > `${b}`
+      ? 1
+      : 0;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultCompare) {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = 0; j < this.length - i - 1; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
+          const temp = this[j];
+
+          this[j] = this[j + 1];
+          this[j + 1] = temp;
```

Comment:
You can shorten your code a little if you swap the values directly
`[this[j], this[j + 1]] = [this[j + 1], this[j]];`